FSQCellManifest/FSQCellRecord.h

```diff
@@ -88,20 +90,20 @@
 @property (nonatomic, copy) NSString *reuseIdentifier;
 
 /**
- This contents of this dictionary are not used internally by the manifest classes. 
+ This contents of this dictionary are not used internally by the manifest classes.
  You can use it to attach arbitrary data to the cell record for your own later use.
  */
-@property (nonatomic, readonly) NSMutableDictionary *userInfo;
+@property (nonatomic, readonly) NSMutableDictionary<NSString *, id> *userInfo;
 
 /**
  Convenience initializer with the most commonly set properties as method parameters
  
  You must include a cellClass. All other parameters are optional.
  */
-- (instancetype)initWithModel:(id)model 
-                    cellClass:(Class)cellClass 
-                  onConfigure:(FSQCellRecordConfigBlock)onConfigure
-                  onSelection:(FSQCellRecordSelectBlock)onSelection;
+- (instancetype)initWithModel:(id)model
```

Comment:
Sooo we do currently allow the model to be nil but I think that this is bad. Basically this means that all of our callbacks that pass model will now need to be marked nullable which will be a pain in the ass to deal with in swift and we'd have to go back through every assume_nonnull class and mark these model methods nullable (yuck!). Maybe we can meet in the middle here and allow the initWithModel to be nullable but if it is nil we set an empty FSQNullModel object internally so we don't have to make every reference to model nullable. Thoughts?